templates/CRM/Admin/Page/PaymentProcessor.hlp

```diff
@@ -293,38 +293,6 @@
 <p>http://checkout.google.com/buttons/checkout.gif?merchant_id=1234567890&amp;w=180&amp;h=46&amp;style=white&amp;variant=text&amp;loc=en_US</p>
 {/htxt}
 
-{htxt id='Moneris-live-user-name'}
-<p>{ts}Moneris doesn't currently use the Username field. Enter 'ignoreme' as a placeholder since the field can't be left blank.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-live-password'}
-<p>{ts}Enter the API Token from your live merchant account.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-live-signature'}
-<p>{ts}Enter your live merchant account Store ID.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-live-url-site'}
-{ts}The URL for the Moneris LIVE Payment server. Use the default value unless otherwise advised by the processor.{/ts}
-{/htxt}
-{htxt id='Moneris-live-url-recur'}
-<p>{ts}The Recurring Payments URL is not currently used for Moneris.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-test-user-name'}
-<p>{ts}Moneris doesn't currently use the Username field. Enter 'ignoreme' as a placeholder since the field can't be left blank.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-test-password'}
-<p>{ts 1='yesguy'}All Moneris test transactions are submitted to a shared test merchant account. Enter <strong>%1</strong> here, which is the API Token for this test account.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-test-signature'}
-<p>{ts 1='store3' 2='store1' 3='store2'}All Moneris test transactions are submitted to a shared merchant account. You can use <strong>%1</strong> as your test store ID. '%2' and '%3' also work, but only '%1' accepts recurring payments.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-test-url-site'}
-{ts}The URL for the Moneris TEST Payment server. Use the default value unless otherwise advised by the processor.{/ts}
-{/htxt}
-{htxt id='Moneris-test-url-recur'}
-<p>{ts}The Recurring Payments URL is not currently used for Moneris.{/ts}</p>
-{/htxt}
```

Comment:
@adixon this help text doesn't seem to be in your extension yet. Can you copy it there?